src/norm_oracles.jl

```diff
@@ -360,3 +360,60 @@ function compute_extreme_point(lmo::EllipsoidLMO, direction; v=nothing, kwargs..
     mul!(v, I, lmo.buffer, -scaling, true)
     return v
 end
+
+
+"""
+    OrderWeightNormLMO(weights,radius)
+    
+LMO with feasible set being the atomic ordered weighted l1 norm: https://arxiv.org/pdf/1409.4271
+
+```
+C = {x ∈ R^n, Ω_w(x) ≤ R} 
+```
+
+"""
+struct OrderWeightNormLMO{W,R,B,D} <: LinearMinimizationOracle
+    weights::W
+    radius::R
+    mat_B::B
```

Comment:
important design: `mat_B` does not need to be a matrix. Indeed, one only ever takes a column, which only has two different values.
So we can instead store the unique non-zero value of B for each element `b`. The number of nonzeros corresponds to the column number, and the dot product of `b_i` with a vector `x` is just:
`sum_{j=1:i} b_val[i] * x[j] == b_val[i] * sum_{j=1:i} x[j]`